.github/workflows/integration_tests.yml

```diff
@@ -115,7 +118,7 @@ jobs:
     # Activate SSH and idle for 30 minutes
 #    - name: Setup tmate session
 #      uses: mxschmitt/action-tmate@v3
-#      timeout-minutes: 30
+#      timeout-minutes: 60
```

Comment:
Nit want to change it back to 30, or update the comment to say 60 minutes?

Reply:
Yes. I missed that during cleanup and set it back to 30 👍